website/layouts/language.erb

```diff
@@ -0,0 +1,1030 @@
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/consul.html">consul</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/cos.html">cos</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/etcd.html">etcd</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/etcdv3.html">etcdv3</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/gcs.html">gcs</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/http.html">http</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/kubernetes.html">kubernetes</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/manta.html">manta</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/oss.html">oss</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/pg.html">pg</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/s3.html">s3</a>
+                  </li>
+                  <li>
+                    <a href="/docs/backends/types/swift.html">swift</a>
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </li><!-- backends -->
+
+          <li>
+            <a href="/docs/configuration/terraform.html">Terraform Settings</a>
+          </li>
+
+        </ul>
+      </li> <!-- configuration blocks -->
+
+
+      <li>
+        <a href="#">Expressions</a>
+        <ul class="nav">
+          <li>
+            <a href="/docs/configuration/expressions/index.html">Overview</a>
+          </li>
+
+          <li>
+            <a href="/docs/configuration/expressions.html">Standard Expressions</a>
+          </li>
+
+          <li>
+            <a href="/docs/configuration/types.html">Type Constraints</a>
+          </li>
+
+          <li>
+            <a href="/docs/configuration/version-constraints.html">Version Constraints</a>
+          </li>
+        </ul>
+      </li>
+
+      <li>
+        <a href="#">Syntax</a>
+        <ul class="nav">
+          <li>
+            <a href="/docs/configuration/syntax/index.html">Overview</a>
+          </li>
+
+          <li>
+            <a href="/docs/configuration/syntax.html">Configuration Syntax</a>
+          </li>
+
+          <li>
+            <a href="/docs/configuration/syntax-json.html">JSON Configuration Syntax</a>
+          </li>
+
+          <li>
+            <a href="/docs/configuration/style.html">Style Conventions</a>
+          </li>
+        </ul>
+      </li>
+
+      <li>
+        <a href="#">Function Reference</a>
```

Comment:
Hmm, that might have been paranoia leaking over from some other decision I was wrestling with. (For example, the old "Modules" section that I renamed to "Module Development," because the shorter title was misleading.) "Functions" is fine! 👍🏼